ui/Modal/ProjectCreation.svelte

```diff
@@ -75,7 +79,7 @@
 
       // Re-fetch session so we have the right permissions to enable the
       // project registration button rithout a page-reload.
-      await fetchSession();
+      fetchSession();
```

Comment:
Lol, not the first time I tripped over the ts event thing not returning a promise. 🤦 

I wonder if we need this at all, as the registry is gone...

Reply:
I think you're right - it doesn't make sense to re-fetch until we actually need to. Will remove it!